app/content/content.ts

```diff
@@ -56,7 +56,7 @@ class Intercept {
 
   setDefaultValues = (responseField, requestsToIntercept, defaultResponseValue) => {
       requestsToIntercept.forEach(req => {
-        if (!(responseField[req.requestId] && responseField[req.requestId].trim())) {
```

Comment:
This change is not enough.
* we need to remove https://github.com/code-mancers/interceptor/pull/81/files#diff-438094ec72049283bd76620ef44fe1d3R70
* Then in onChange event of textarea,